### ISSUE TITLE
Add a link on the menu bar to the new status page

### ISSUE THREAD

**Issue by kineticsquid** [EclipseFdn/open-vsx.org#1295]
We should have a `Status` link on the menu bar around `Documentation` ... `Sponsor` that points to our new open-vsx.org status page at status.open-vsx.org.

**Comment by kineticsquid** [comment]
@amvanbaren FYI

**Comment by kineticsquid** [comment]
OK, re-opening this because of my oversight. 

First, @scar45 @yiningwang11, thanks both of your for your PRs implementing this enhancement.

@amvanbaren thanks for pointing out the duplication. I accidentally merged two different PRs solving this problem, https://github.com/EclipseFdn/open-vsx.org/pull/1428 and https://github.com/EclipseFdn/open-vsx.org/pull/1440. I got tangled up when I attempted to merge one of these using the wrong ID which caused it to fail, back it out, and retry (by accident with the other PR). Sigh...

@chrisguindon Helped me untangle this and re-do the PR: https://github.com/EclipseFdn/open-vsx.org/pull/1461. Ready for review and merge.

**Comment by kineticsquid** [comment]
OK, re-opening this because of my oversight. 

First, @scar45 @yiningwang11, thanks both of your for your PRs implementing this enhancement.

@amvanbaren thanks for pointing out the duplication. I accidentally merged two different PRs solving this problem, https://github.com/EclipseFdn/open-vsx.org/pull/1428 and https://github.com/EclipseFdn/open-vsx.org/pull/1440. I got tangled up when I attempted to merge one of these using the wrong ID which caused it to fail, back it out, and retry (by accident with the other PR). Sigh...

@chrisguindon Helped me untangle this and re-do the PR: https://github.com/EclipseFdn/open-vsx.org/pull/1461. 

Ready for review and merge.
